modules/seqexec/engine/src/test/scala/seqexec/engine/StepSpec.scala

```diff
@@ -197,7 +196,7 @@ class StepSpec extends FlatSpec {
           (seqId,
             Sequence.State.Zipper(
               Sequence.Zipper(
-                id = "First",
+                id = Observation.Id.unsafeFromString("GS-2018A-Q-3"),
```

Comment:
Yeah just to avoid confusion I would use a normal id here and elsewhere.

```scala
scala> gem.Observation.Id.unsafeFromString("GS-2018A-Q-3") // works, but weird
res10: gem.Observation.Id = Id(Nonstandard(Some(GS),Some(Semester(2018A)),None,Q),Index(3))

scala> gem.Observation.Id.unsafeFromString("GS-2018A-Q-3-123")
res11: gem.Observation.Id = Id(Science(GS,Semester(2018A),Q,Index(3)),Index(123))
```

You could also generate a random one.

```scala
scala> gem.arb.ArbObservation.arbObservationId.arbitrary.sample.get
res12: gem.Observation.Id = Id(Daily(GN,ENG,2017-09-20),Index(97))
```